andela/phlox-ah
# #161141274 check if user is verified

#### What does this PR do?
It checks if a user is verified before logging in the user

#### Description of Task to be completed?
- create a condition that checks if the user has verified their account
- write a test to test the condition

#### How should this be manually tested?
- Sign up and try logging in without verifying your account it will throw an error
- Verify your account and try logging in, you will be logged in

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
Story type: feature
Story title: A user should verify their account before they can log in and perform any other authenticated operation.
Story ID: #161141274

#### Screenshots (if appropriate)
N/A
#### Questions:```
N/A
